elastic/kibana
# [Reporting] Remove banner from reports

## Summary

Partially addresses https://github.com/elastic/kibana/issues/116103

We make sure to not render banners from the UI in reports. For now this just removes banners from Visualize Library PDF/PNG reports due to how the vis was being repositioned when capturing a screenshot.

## How to test

1. Activate banners on your trial licensed Kibana
2. Go to Visualize and request a report
3. Check the resulting report, no banner should be included

## Release note

We now prevent banners from unintentionally appearing in Reports and possibly hiding content.


### Checklist


### Future work

We should, in future work consider adding the ability to add custom banners to reports too (see https://github.com/elastic/kibana/issues/116103#issuecomment-950936604).
